### DISCUSSION TITLE
Regarding official communication channel for gsoc 2023 participants

### DISCUSSION THREAD

**Discussion by RaviRahar** [kworkflow/kworkflow#816] [Q&A] [answered]
I came here from the `kworkflow` mentioned in `linux-foundations'` project ideas of 2023 gsoc. I have gone through the tutorial till kernel compilation. I am now to start working with kworkflow.  
But I do not know the official communication channel. I found these two:
- The gsoc page mentions `#kworkflow` on `irc.freenode.net`. But as far as I know  freenode is dead? Is this channel still in use? 
- The [how to contribute](https://kworkflow.org/content/howtocontribute.html#) page mentions `#kw-devel` channel on `OFTC`. I contacted there but no one replied. Also irc was new for me, so I did not know I have to be always online to receive messages. So after gaining this knowledge, I contacted again and no one replied.

I wanted an ID from @rodrigosiqueira  for the assignments mentioned in proposal. And also what is the official communication channel?

**Comment by rodrigosiqueira** [comment, marked as answer]
Hi,

> I came here from the `kworkflow` mentioned in `linux-foundations'` project ideas of 2023 gsoc. I have gone through the tutorial till kernel compilation. I am now to start working with kworkflow. But I do not know the official communication channel. I found these two:
> 
>     * The gsoc page mentions `#kworkflow` on `irc.freenode.net`. But as far as I know  freenode is dead? Is this channel still in use?

We use OFTC:

https://kworkflow.org/content/howtocontribute.html#reach-out

If you find some reference to Freenode, please consider sending a patch fixing that.

>     * The [how to contribute](https://kworkflow.org/content/howtocontribute.html#) page mentions `#kw-devel` channel on `OFTC`. I contacted there but no one replied. Also irc was new for me, so I did not know I have to be always online to receive messages. So after gaining this knowledge, I contacted again and no one replied.

I recommend you read:

https://en.wikipedia.org/wiki/Internet_Relay_Chat

Keep in mind that IRC is 'stateless', i.e., if you do not have something like IRC bouncer (e.g., ZNC), nobody will be able to reach you as soon as you log out. So, if you send a message in the channel and log out, nobody will be able to reply to you.

Finally, do not expect a real-time reply on IRC; I don't have it on my phone (I only access it on some specific devices). Sometimes I'll follow IRC very close, but sometimes not. Aside from IRC, you can always use Github Issues, Discussion, and emails.
 
>
> I wanted an ID from @rodrigosiqueira for the assignments mentioned in proposal. And also what is the official communication channel?

Please, send me an email about that. If, for some reason, I missed your email, don't be afraid to ping me again.

Thanks

**Reply by RaviRahar** [reply]
Okay, I have sent the mail. Please check.
